crates/api_models/src/enums.rs

```diff
@@ -1025,6 +1025,20 @@ pub struct UnresolvedResponseReason {
 #[serde(rename_all = "snake_case")]
 #[strum(serialize_all = "snake_case")]
 pub enum FieldType {
+    UserFullName,
+    UserEmailAddress,
+    UserPhoneNumber,
+    UserCountry { options: Vec<String> },
+    UserAddressline1,
+    UserAddressline2,
+    UserAddressCity,
+    UserAddressPincode,
+    UserAddressState,
+    UserAddressCountry,
+    UserBlikCode,
+    FieldsComplete,
+    UserBillingName,
+    UserBank,
```

Comment:
`FieldType` should only specify the type of the required field like text or dropdown. because it makes the sdk team to identify the type easily rather than particularly specifying like `UserFullName`. We are anyways providing this info in `display` field right. what do u think?

Reply:
Sdk team asked for this change specifically.

Reply:
ok then fine